dispatch/tests/test_api_pages.py

```diff
@@ -0,0 +1,226 @@
+from django.core.urlresolvers import reverse
+
+from rest_framework import status
+
+from dispatch.tests.cases import DispatchAPITestCase
+
+from dispatch.apps.content.models import Page
+
+class PagesTest(DispatchAPITestCase):
+
+    def _create_page(self):
+        """
+        Create Dummy Page
+        """
+
+        url = reverse('api-pages-list')
+
+        data = {
+          "title": "Test Page",
+          "slug": "test-page",
+          "snippet": "This is a test snippet",
+          "content": [
+            {
+              "type": "paragraph",
+              "data": "This is some paragraph text"
+            }
+          ]
+        }
+
+        return self.client.post(url, data, format='json')
+
+    def test_create_page_unauthorized(self):
+        """
+        Create page while unauthorized should fail with unauthenticated request
+        """
+        # Clear authentication credentials
+        self.client.credentials()
+
+        url = reverse('api-pages-list')
+
+        data = {
+          "title": "Test Page",
+          "slug": "test-page",
+          "snippet": "This is a test snippet",
+          "content": [
+            {
+              "type": "paragraph",
+              "data": "This is some paragraph text"
+            }
+          ]
+        }
+
+        response = self.client.post(url, data, format='json')
+
+        self.assertEqual(response.status_code, status.HTTP_401_UNAUTHORIZED)
+
+    def test_create_empty_page(self):
+        """
+        Create page should fail with empty payload
+        """
+
+        url = reverse('api-pages-list')
+
+        self._create_page()
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_create_incomplete_page(self):
+        """
+        Creat Page should fail with missing required fields
+        """
+
+        url = reverse('api-pages-list')
+
+        # Page missing content, snippet, slug
+        data = {
+            "title": "Test Page",
+        }
+
+        response = self.client.post(url, data, format='json')
+
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+        # Check Data
+        self.assertTrue('content_json' in response.data)
```

Comment:
Sorry, forgot to mention that the `content_json` field has been renamed to just `content`